pkg/controller/packageregistry/pod.go

```diff
@@ -0,0 +1,140 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package packageregistry
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	eprv1alpha1 "github.com/elastic/cloud-on-k8s/v3/pkg/apis/epr/v1alpha1"
+	"github.com/elastic/cloud-on-k8s/v3/pkg/controller/common/certificates"
+	"github.com/elastic/cloud-on-k8s/v3/pkg/controller/common/container"
+	"github.com/elastic/cloud-on-k8s/v3/pkg/controller/common/defaults"
+	"github.com/elastic/cloud-on-k8s/v3/pkg/controller/common/metadata"
+	"github.com/elastic/cloud-on-k8s/v3/pkg/controller/common/version"
+)
+
+const (
+	HTTPPort                 = 8080
+	configHashAnnotationName = "packageregistry.k8s.elastic.co/config-hash"
+	EPRTLSKey                = "EPR_TLS_KEY"
+	EPRTLSCert               = "EPR_TLS_CERT"
+	EPRAddress               = "EPR_ADDRESS"
```

Comment:
nit: we are already in a package named `packageregistry`, we can remove the `EPR` prefix:
```suggestion
	TLSKeyEnvName            = "EPR_TLS_KEY"
	TLSCertEnvName           = "EPR_TLS_CERT"
	AddressEnvName           = "EPR_ADDRESS"
```